Example/Highlightr.xcodeproj/project.pbxproj

```diff
@@ -619,6 +502,7 @@
 			baseConfigurationReference = FACBE1ED06E5718AE417DCC0 /* Pods-Highlightr_Example.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = RV92JH4QP5;
```

Comment:
Please remove your development team

Reply:
Hi! I thought I had removed it but forgot to do so in the Example Project. Should be fixed now.

03dd0f36334ee12d8215e52290b1fc617d255b80